### ISSUE TITLE
android client. user and channel items are defined as read only objects for talkback, but they should be clickable

### ISSUE THREAD

**Issue by amirmahdifard** [BearWare/TeamTalk5#2514]
### Description

talkback has 2 type of feadback sounds, that when focusing things on the screen, it will let user know the selected option is just a read only text, or an option that can be activated and be clicked, and be interacted with. Currently, talkback knowns the user and channel options as read only, but they are actually not. So this misleading should be resolved. please don't close this issue, I assighn this to my self and will fix it soon

### Application

- [ ] qtTeamTalk
- [X] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkServer

**Comment by bear101** [comment]
Do you plan to fix this? Otherwise I'll move it to Discussions' feature requests.

**Comment by amirmahdifard** [comment]
@bear101 yes, I plan to fix this
